interview-questions.md

```diff
@@ -9,59 +9,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The keyword "this" refers to the object that is currently or specifically being invoked in a function.
 
-  Researched answer:
+  Researched answer: When calling a function as a method of the object, using "this" refers to that specific object being used or being referred to. 
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a Javascript library for building user interfaces. React contains code snippet that is reusable. It is component based. 
 
-  Researched answer:
+  Researched answer: React is a front-end Javascript Library that is used for building user interfaces. It can be used to create web applications and mobile apps. Because React is a library, it has reusuable code snippers and other UI components that allows it to be usuable. 
```

Comment:
Perfect!  As developers, we love to keep our code DRY (Don't Repeat Yourself), and React helps us do that with reusable components.  We also get to utilize the benefits of the virtual DOM speeding up development time and user experience as well.